### DISCUSSION TITLE
Pull request for ideas listed at end of ReadMe

### DISCUSSION THREAD

**Discussion by kenminsoo** [rahulnyk/knowledge_graph#21] [Ideas]
Is this repo still open to pull requests to potentially implement ideas at end of readme? I was thinking of forming a small team to do so since the questions seem interesting. Has any work been done on these? 

Cool idea! Instead of reinventing the wheel, its cooler to make this one more shiny I think.

**Comment by rahulnyk** [comment]
Yes very much :)
